crypto/bls/mask.go

```diff
@@ -75,7 +75,7 @@ type Mask struct {
 // cosigners are disabled by default. If a public key is given it verifies that
 // it is present in the list of keys and sets the corresponding index in the
 // bitmask to 1 (enabled).
```

Comment:
Can you update the documentation to remove the content `If a public key is given...` ?